distro/archlinux/PKGBUILD

```diff
@@ -2,7 +2,7 @@
 # Contributor: xatier
 _pkgname=fcitx5-mcbopomofo
 pkgname=fcitx5-mcbopomofo-git
-pkgver=2.5.2.r16.g8f629e9
+pkgver=2.5.2.r176.gf1aab54
```

Comment:
Isn't this pointing to the current HEAD? Just wanted to make sure this your intention.

Reply:
Yes, this is more like a placeholder for the package version, it's automatically bumped by the `makepkg` tool. For `-git` AUR packages (packages that aim for a HEAD build), typically we point to the HEAD commit. This is done by the `pkgver` function below. 

Ref:

https://wiki.archlinux.org/title/VCS_package_guidelines#The_pkgver()_function
https://gitlab.archlinux.org/pacman/pacman/blob/master/proto/PKGBUILD-vcs.proto#L33